doc/nrf/releases/release-notes-1.5.1.rst

```diff
@@ -0,0 +1,79 @@
+.. _ncs_release_notes_151:
+
+|NCS| v1.5.1 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status. See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Added Radio front-end module (FEM) support for Bluetooth LE in single and multiprotocol use cases
```

Comment:
This highlight is missing in the changelog.

Reply:
Added after an offline request. @kromancer

Reply:
Most of the changes since 1.5 are captured in the changelogs from sdk-nrfxlib:
https://github.com/nrfconnect/sdk-nrfxlib/blob/v1.5-branch/softdevice_controller/CHANGELOG.rst#added

Reply:
Added a link to the sdk-nrfxlib changelog in the changelog section of the release notes.